### PR TITLE
pkp/pkp-lib#9736 Allow ordering of masthead roles

### PR DESCRIPTION
**Pull request by bozana**
s. https://github.com/pkp/pkp-lib/issues/9736

### CODE REVIEW

File: locale/se/manager.po [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Huw Robert Grange <huw.r.grange@uit.no>, 2023.
+msgid ""
+msgstr "X-Generator: Weblate\nMIME-Version: 1.0\nContent-Type: text/plain; charset=UTF-8\nContent-Transfer-Encoding: 8bit"
```
Comment (asmecher) [inline locale/se/manager.po:3]: It looks like you intended to add a translation here but there's just a bit of metadata.

**Review by asmecher — CHANGES_REQUESTED** [review]
Just a couple of minor comments, thanks!

File: locale/sq/manager.po [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+"Language: sq\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 4.13.1\n"
+
+
+msgid "manager.setup.masthead"
+msgstr ""
```
[the remark this answers is not in the document]
Comment (bozana) [reply locale/sq/manager.po:17]: Yes, I moved this key from OJS to pkp-lib and it was also empty in the OJS. So I will leave it as it is...

File: locale/se/manager.po (hunk 1 above)
Comment (bozana) [reply locale/se/manager.po:3]: I will then remove this file. It happened when using the CLI tool to move a locale key from OJS into PKP-LIB...
I haven't notices this. Thanks!

**Pull request merged**